import-service/serverless.yml

```diff
@@ -8,6 +8,17 @@ provider:
   stage: dev
   region: eu-west-1
 
+  environment:
+    SQS_URL:
+      Ref: SQSQueue
+    SNS_ARN:
+      Ref: SNSTopic
+    PG_HOST: lesson4-instance.c3zxepihdznj.eu-west-1.rds.amazonaws.com
+    PG_PORT: 5432
+    PG_DATABASE: lesson4
+    PG_USERNAME: postgres
+    PG_PASSWORD: wwIJ7CchntfGgnOsOzxk
```

Comment:
hardcoded credentials - should be hidden - for example in .env file

Reply:
+
Please, consider hiding sensitive data (emails/passwords/etc) into `.env` files